scripts/flytekit_build_image.sh

```diff
@@ -16,8 +16,15 @@ echo "           DOCKER BUILD"
 echo "------------------------------------"
 echo ""
 
+if [ -z "$DOCKERFILE_PATH" ]; then
```

Comment:
Ohh my god!, we should basically convert this to a python script.

Reply:
Also it might be better to use Flags - https://www.baeldung.com/linux/use-command-line-arguments-in-bash-script#flags

Reply:
i actually feel python is way more complicated.  once you do that, you tend to want to bring in things like click, have to use subprocess, and then you start seeing python stack traces, etc.  I'd rather get rid of this script entirely if possible, and leave as is if not.